app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -70,9 +70,10 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="32dp"
             android:contentDescription="@string/required_fields_content_description"
-            android:text="@string/add_profile_required"
+            android:text="@{'*' + @string/add_profile_required}"
```

Comment:
This should probably be done in Kotlin, instead, to ensure we are formatting it correctly (we have a lot of bidirectional wrapping stuff coming for proper localization, and this is a case that may be easily missed).

Also, is there a test for this case?